src/Donations/PaymentMethods/BankTransfer.tsx

```diff
@@ -18,7 +18,7 @@ function BankTransfer({ onSubmitPayment }: Props): ReactElement {
 
       <button
         className="primary-button w-100 mt-30"
-        onClick={() => onSubmitPayment("offline", "offline")}
+        onClick={() => onSubmitPayment("offline", "transfer")}
```

Comment:
@jmiridis Should we send `method = transfer` or `method = offline` within the `paymentProviderRequest `while calling `PUT /app/donations/:donationId`?

Reply:
Assuming that the 1st parameter is the gateway/provider and the 2nd parameter is the payment method, `onSubmitPayment("offline", "transfer")` is the correct one.